osgeo_importer/models.py

```diff
@@ -150,22 +150,20 @@ def filesize(self):
         """
         Humanizes the upload file size.
         """
-        if not self.size:
-            uploaded_file = self.uploadfile_set.first()
-
-            if uploaded_file:
-                self.size = uploaded_file.file.size
-                self.save()
-            else:
-                return
-
+        size = 0
+        if not self.size or self.size == 0:
+            if self.uploadfile_set.count() > 0:
+                for uf in self.uploadfile_set.all():
+                    size += uf.file.size
+        self.size = size
+        self.save()
         return sizeof_fmt(self.size)
 
     def file_url(self):
         """
         Exposes the file url.
         """
-        return self.uploadfile_set.first().file.url
+        return '' # self.uploadfile_set.first().file.url
```

Comment:
Where is this used? I'd personally rather remove the method and in that way cleanly break existing API than start returning something which may cause silent or confusing failures.